src/configuration/appDefaults.js

```diff
@@ -0,0 +1,16 @@
+const DASHBOARD_CONFIG = {
+  landingPath: '/win10/overview?numDays=90',
+  timeRange: 90, // 90 days
+  colors: ['#e55525', '#ffcd02'],
+  labels: ['Firefox', 'Chrome'],
+};
+
+export const SETTINGS = {
+  frameworks: {
+    TALOS_FRAMEWORK_ID: 1,
```

Comment:
At this point simply define `talos` as the key.
The upper case naming is a convention when defining constant.
Since the `talos` key would be insider of the `frameworkIds` key is unnecessary to name the key like `talos_framework_id` since accessing the value would look like this `DASHBOARD_CONFIG.frameworkIds.talos` and have the word 'framework' in there.